### ISSUE TITLE
Cannot find and install deb package via apt-get

### ISSUE THREAD

**Issue by wli3** [dotnet/cli#6796]
1. Have a clean machine from official ubuntu iso/ Azure VM
2. Run following command, to add our _source_ 

``` shell
sudo sh -c 'echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'

sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893

sudo apt-get update
```
3. 
```
apt-cache search dotnet-sdk | grep 2.0.0
```

Expect there is package and can be installed, however there is nothing right now

**Comment by wli3** [comment]
That package issue might be cause of master build failure, Ubuntu 14.04 16.04 and Debian failed last time with new naming change, I am requeueing

**Comment by wli3** [comment]
Ubuntu 16.10 is good, which strangely has both correct runtime version and last CLI/SDK build succeeded

**Comment by wli3** [comment]
- [ ] https://github.com/dotnet/core-setup/issues/2657
- [ ] Update runtime version after new build it shows up in the feed
- [ ] Fix VSTS release 2.0.0 build

**Comment by wli3** [comment]
Tested on 16.04, I can install it via apt-get now
![image](https://user-images.githubusercontent.com/6993335/27107628-f1133b02-504d-11e7-9f9b-97bb5942c3cc.png)
